app/mailers/order_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class OrderMailer < ApplicationMailer
+  def order_confirm order
+    @order = order
+    User.admin.each do |f|
+      mail to: f.email, subject: t("mailers.order.order_confirm")
```

Comment:
mail gửi tới tập thể admin luôn nha e, gửi từng người từng cái như này performance ko tốt
mail to: User.admin.pluck(:email)